content/en/developers/metrics/custom_metrics.md

```diff
@@ -3,226 +3,244 @@ title: Custom Metrics
 
-{{< img src="developers/metrics/custom_metrics/metric_summary.png" alt="metric_summary" responsive="true" style="width:70%;">}}
+Adding a second host with 3 services and 1 status = **9 distinct metrics**:
 
-Adding the second host:
+{{< img src="developers/metrics/custom_metrics/metric_summary_2.png" alt="metric_summary_2" responsive="true" style="width:80%;">}}
 
-{{< img src="developers/metrics/custom_metrics/metric_summary_2.png" alt="metric_summary_2" responsive="true" style="width:70%;">}}
+Adding a third host with 2 services and 2 status = **13 distinct metrics**:
 
-Adding the third host as per the table above, you get your 13 distinct metrics:
+{{< img src="developers/metrics/custom_metrics/metric_summary_3.png" alt="metric_summary_3" responsive="true" style="width:80%;">}}
 
-{{< img src="developers/metrics/custom_metrics/metric_summary_3.png" alt="metric_summary_3" responsive="true" style="width:70%;">}}
+#### Query editor
 
-Using the query editor, you can also find this using the count: aggregator
+You can count your custom metrics by using the `count:` aggregator in the query editor. For counting the previous example, the query `count:service.request.count{*}` is used:
 
-{{< img src="developers/metrics/custom_metrics/metric_aggregator.png" alt="metric_aggregator" responsive="true" style="width:70%;">}}
+{{< img src="developers/metrics/custom_metrics/metric_aggregator.png" alt="metric_aggregator" responsive="true" style="width:80%;">}}
 
-Ultimately, you'll have 13 metrics using the following query: `count:service.request.count{*}`
+{{< img src="developers/metrics/custom_metrics/count_of_metrics.png" alt="count_of_metrics" responsive="true" style="width:80%;">}}
 
-{{< img src="developers/metrics/custom_metrics/count_of_metrics.png" alt="count_of_metrics" responsive="true" style="width:70%;">}}
+### Gauges, counts, histograms, and rates
 
-### Counting custom metrics from gauges, counts, histograms, and rates
-A [gauge][9] represents one value per second (examples: temperature or Kafka queue offset).
+Suppose you are interested in measuring the average `temperature` in the state of Florida. `temperature` is stored as a [GAUGE][10] metric type in Datadog. You collect the following temperature measurements every 10 seconds during the past minute from Orlando, Miami, Boston, New York, and Seattle, each tagged with information about the `city`, `state`, `region`, and `country`.
 
-Suppose you are interested in measuring the average `temperature` in the state of Florida. `temperature` is stored as a `gauge` metric type in Datadog. You collect the following temperature measurements every 10 seconds during the past minute from Orlando, Miami, Boston, New York and Seattle, each tagged with information about the `city`, `state`, `region`, and `country`.
+| Tags                         | Temp 1 | Temp 2 | Temp 3 | Temp 4 | Temp 5 | Temp 6 | Temp 7 |
```

Comment:
Temp1...Temp 7 confuses me since intuitively I'd assume it's only 6 measurements in a minute. Can we replace with a 0-10s, 10s-20s, etc. timeline?